core/src/com/unciv/logic/civilization/ReligionManager.kt

```diff
@@ -152,6 +150,13 @@ class ReligionManager : IsPartOfGameInfoSerialization {
         }
     }
 
+    fun numberOfBeliefsAvailable(type: BeliefType): Int {
+        return civInfo.gameInfo.ruleSet.beliefs.values.count {
+            it.type == type
+                    && civInfo.gameInfo.religions.values.none { religion -> it in religion.getBeliefs(type) }
+        }
```

Comment:
If you just want to raw number, it might even be faster to count the total number of beliefs of a certain type, and then subtract the number of beliefs each civ has of that type, but that's a minor optimization